### PR TITLE
add master and system integration for clearbit

### PR DESCRIPTION
**Pull request by niranjan92**
https://github.com/Shippable/base/issues/1172

### CODE REVIEW

File: versions/master.json [hunk 1]
```diff
@@ -244,6 +244,11 @@
       ]
     },
     {
+      "name": "clearbitKeys",
+      "type": "generic",
+      "services" : []
```
Comment (shrivara) [inline versions/master.json]: Where are we going to use the keys?

Comment (niranjan92) [inline versions/master.json]: @shrivara : ClearbitAdapter is called from micro, hubspotSync/_eventHandlers/signupEvent.js for enriching user information before sending to hubspot.

Comment (niranjan92) [inline versions/master.json]: there is no separate service that uses it.

File: migrations/master.sql [hunk 2]
```diff
@@ -1493,6 +1493,18 @@ do $$
...
 
+    -- clearbitKeys master integration
+    if not exists (select 1 from "masterIntegrations" where "name" = 'clearbitKeys' and "typeCode" = 5012) then
+      insert into "masterIntegrations" ("id", "masterIntegrationId", "name", "displayName", "type", "isEnabled", "level", "typeCode", "createdBy", "updatedBy", "createdAt", "updatedAt")
+      values ('58c78481e34468d32114e125', 58, 'clearbitKeys', 'Clearbit Keys', 'generic', false, 'generic', 5012, '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-03-14', '2017-03-14');
+    end if;
+
+    -- masterIntegrationFields for clearbitKeys
+    if not exists (select 1 from "masterIntegrationFields" where "id" = 194) then
+      insert into "masterIntegrationFields" ("id", "masterIntegrationId", "name", "dataType", "isRequired", "isSecure","createdBy", "updatedBy", "createdAt", "updatedAt")
```
Comment (shrivara) [inline migrations/master.sql]: Space after `"isSecure", `

**Pull request merged**